Jenkinsfile

```
@@ -82,7 +80,7 @@ void runTest(String TEST_NAME, String CLUSTER_PREFIX) {
                     else
                         export KUBECONFIG=/tmp/$CLUSTER_NAME-${CLUSTER_PREFIX}
                         source $HOME/google-cloud-sdk/path.bash.inc
-                        ./e2e-tests/$TEST_NAME/run
+                        time bash ./e2e-tests/$TEST_NAME/run
```

```suggestion
                         ./e2e-tests/$TEST_NAME/run
```
Please remove it because for now it looks like you forgot to delete debug. If you want to have this stat (but I think it is not needed at all) you need to add the column into report in PR https://github.com/percona/percona-xtradb-cluster-operator/pull/838#issuecomment-823073154 and save this info into S3 bucket to have history.